simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.xml

```diff
@@ -9,6 +9,7 @@
 
   <!-- model parameters -->
   <arg name="simulator_model" default="$(find-pkg-share simple_planning_simulator)/config/simple_planning_simulator.param.yaml"/>
+  <arg name="vehicle_param_file"/>
```

Comment:
This argument is not given by vehicle_interface.launch.xml.

Reply:
It corresponds with this PR ( https://github.com/tier4/autoware_launcher.iv.universe/pull/64/commits/3c79c56fa5f381a1142351a0f81fba187e153572 )

Reply:
Sorry, I forgot to need a build for the launch file.